sourcegraph/about
# Redesign old Guide pages

This closes #5370. It reformats the remaining two Guide pages to conform to our new design standard.

### Changelog

- Implements the `GatedResourceLayout` on `/guides/continuous-developer-onboarding` and `/guides/dev-onboarding-how-is-it-unique`
- Prettier update

### Notes
- The copy in the Figma differs from what is in production. Fabiana comments that we should keep the copy that is in production.
- For the HubSpot forms, I still need to remove the titles ("Download the guide") and extend the width to 100%. I am waiting until eng review is complete to make these changes, as these same forms are in production!

Example of a form to be updated (duplicated title, not full-width):
<img width="400" alt="Screen Shot 2022-06-08 at 4 27 17 PM" src="https://user-images.githubusercontent.com/60713139/172720445-90d38593-1efe-47aa-9d95-08ac431b51bc.png">

Needs to match the correct format:
<img width="400" alt="Screen Shot 2022-06-08 at 4 35 41 PM" src="https://user-images.githubusercontent.com/60713139/172721525-e7d17c5f-e3ba-4558-b329-9da48e0f2e76.png">


### Test

1. Ensure prettier has standardized the proposed changes.
2. Navigate to `/guides/continuous-developer-onboarding`
3. Navigate to `/guides/dev-onboarding-how-is-it-unique`
4. Please check that the PDFs download on both pages after form submission
